index.bs

```diff
@@ -190,6 +190,9 @@ the user agent MUST run the following steps:
     {{leavepictureinpicture}} at the |video| with its {{bubbles}} attribute
     initialized to true.
 
+The user agent SHOULD NOT change video playback state when video leaves
+Picture-in-Picture.
```

Comment:
I think this should only apply when the `exitPictureInPicture()` method is called. As defined here, it would apply to *all* ways of leaving Picture-in-Picture, right?

Reply:
I can be more specific and say something like "The user agent SHOULD NOT change video playback state when the exit Picture-in-Picture algorithm is run".
This means it would apply only to when the `exitPictureInPicture()` method is called and when `disablePictureInPicture` is set on an existing PiP video. What do you think?

Reply:
That SGTM. Though, I would consider using NOT RECOMMENDED just because technically, changing the playback state wouldn't follow the algorithm already. It may be interesting to give more info, explaining that it may be okay to have side effects from interacting with a UI element but if it's website initiated, the website should be in control of the experience.

Reply:
What do you think of https://github.com/WICG/picture-in-picture/pull/73/commits/f0c6e707b7d032b0248e711054cee2e9381c3942?